docs/user-guide/connectors/twilio.rst

```diff
@@ -29,6 +29,11 @@ You need to set up a Twilio account.
      steps to connect a phone number to the project.
   3. Now you can use the ``Account SID``, ``Auth Token``, and the phone
      number you purchased in your ``credentials.yml``.
+  4. Configure your webhook URL by navigating to 
+     [Phone Numbers](https://www.twilio.com/console/phone-numbers/incoming) in the Twilio 
+     dashboard and selecting your phone number. Find the ``Messaging`` section and add 
+     your webhook URL (e.g. ``https://yyyyyy.ngrok.io/webhooks/twilio/webhook``) to the 
```

Comment:
why ngrok?

Reply:
Good question -- that is what karen put, likely from when she was trying it out. What aabout something like 
```suggestion
     your webhook URL (e.g. ``https://<host>:<port>/webhooks/twilio/webhook``) to the 
```
and then adding (`replacing the host and port with your running Rasa X or Rasa Open Source server`)

Reply:
Sounds good to me. That is more clear